docs/develop/json-manifest-overview.md

```diff
@@ -0,0 +1,871 @@
+
+  ```json
+  "items": [
+      {
+          -- markup for a menu item is here --
+      },
+      {
+          -- markup for another menu item is here --
+      }
+  ]
+  ```
+
+#### Top level structure
+
+The root level of the preview JSON manifest, which roughly corresponds to the **OfficeApp** element in the current XML manifest, is an anonymous object. 
+
+The children of **OfficeApp** are commonly divided into two notional categories. The **VersionOverrides** element is one category. The other consists of all the other children of **OfficeApp** which are collectively referred to as the base manifest. So too, the preview JSON manifest has a similar division. There is a top-level "extension" property that roughly corresponds in its purposes and child properties to the **VersionOverrides** element. The preview JSON manifest also has over 10 other top-level properties that collectively serve the same purposes as the base manifest of the XML manifest. So, these other properties can be thought of collectively as the base manifest of the JSON manifest. 
+
+> [!NOTE]
+> When it becomes possible to combine an add-in with other Microsoft 365 extension types in a single web app, then there will be other top-level properties that don't fit into the notion of the base manifest. Roughly speaking, there will be a top-level property for every kind of Microsoft 365 extension type, such as "configurableTabs", "bots" and "connectors". For examples, see the [Teams manifest documentation](/microsoftteams/platform/resources/schema/manifest-schema). This structure makes clear that the "extension" property represents an Office add-in as one type of Microsoft 365 extension.
+
+#### Base manifest
+
+The base manifest properties specify characteristics of the add-in that *any* type of extension of Microsoft 365, including Teams tabs or message extensions, would be expected to have, not just Office add-ins. These characteristics include a public name and a unique ID. The following table shows a mapping of some critical top level properties in the preview JSON manifest to the XML elements in the current manifest, where the mapping principle is the *purpose* of the markup.
+
+|JSON property|Purpose|XML element(s)|Comments|
+|:-----|:-----|:-----|:-----|
+|"$schema"| Identifies the manifest schema. | attributes of **OfficeApp** and **VersionOverrides** | |
+|"id"| GUID of the add-in. | **Id**| |
+|"version"| Version of the add-in. | **Version** | |
+|"manifestVersion"| Version of the manifest schema. |  attributes of **OfficeApp** | |
+|"name"| Public name of the add-in. | **DisplayName** | |
+|"description"| Public description of the add-in.  | **Description** | |
+|"accentColor"||| This property has no equivalent in the current XML manifest and is not used in the preview of the JSON manifest. But it must be present. |
```

Comment:
For me, the takeaway is that you need to add this, but it does nothing. I don't think more needs to be said.
```suggestion
|"accentColor"||| Required, but isn't supported in the preview. |
```

Reply:
I prefer it as is. This table is meant to orient someone familiar with the XML manifest, by loosely mapping elements to properties. It is not reference documentation for the properties.